lib/desired-caps.js

```diff
@@ -33,6 +33,9 @@ let desiredCapConstraints = _.defaults({
   tapWithShortPressDuration: {
     isNumber: true
   },
+  scaleFactor: {
+    isString: true
```

Comment:
I've implemented value validation in https://github.com/appium/appium-ios-simulator/pull/74  :

```javascript
 setScaleFactor (newScaleFactor) {
    let supportedScales = ['1.0', '0.75', '0.5', '0.33', '0.25'];
    if (supportedScales.indexOf(newScaleFactor) < 0) {
      let msg = `Only "${supportedScales}" values are supported as scale factors. "${newScaleFactor}" is passed instead.`;
      log.errorAndThrow(msg);
    }
    this.scaleFactor = newScaleFactor;
  }
```